support/cas-server-support-couchdb-core/src/main/resources/org/apereo/cas/couchdb/CAS_delete_record.js

```diff
@@ -0,0 +1,8 @@
+function(doc, req){
```

Comment:
Should this be removed?

Reply:
Probably... it's not used by anything but i left it there as a starter for other stuff...

Reply:
Done